packages/block-template/webpack-dev-server.config.js

```diff
@@ -2,7 +2,9 @@ const HtmlWebpackPlugin = require("html-webpack-plugin");
 const webpack = require("webpack");
 const config = require("./webpack.config");
 
+/** @type import("webpack").Configuration */
 module.exports = {
+  devtool: "eval-cheap-module-source-map",
```

Comment:
Replaces `-d` option` in `package.json` → `scripts`→ `... webpack-dev-server ...`. The new version of the server requires a value after `-d` (or `--devtool`), so leaving the command as is produces an exception.